shared/src/main/scala/coop/rchain/shared/StringOps.scala

```diff
@@ -4,10 +4,16 @@ import scala.util.Try
 
 object StringOps {
   implicit class StringColors(str: String) {
-    def green: String     = s"\u001B[32m" + str + "\u001B[0m"
-    def red: String       = s"\u001B[31m" + str + "\u001B[0m"
-    def blue: String      = s"\u001B[34m" + str + "\u001B[0m"
-    def isNumber: Boolean = str.matches("[+-]?\\d+.?\\d+")
+    private def conditionalColor(coloring: String)(implicit tty: Boolean): String =
```

Comment:
Boolean is not specific enough to be used as an implicit parameter. We either need a custom value type here, or some type tag, or maybe the `TerminalMode` capability itself.